backend/vouch/trust_algo.py

```diff
@@ -9,39 +9,42 @@
 
 logger = logging.getLogger(__name__)
 
-# In this algorithm, we leverage pretrust (e.g., based on email domains) and vouching
-# to securely assign voting rights to a wider set of contributors.
-# The algorithm inputs pretrust status and a vouching directed graph.
+# In this algorithm, we leverage pre-trust (e.g., based on email domains) and
```

Comment:
Here `pre-trust` refers to pre-trusted users or to something else?

Reply:
I would propose to rephrase "pre-trust" to "pre-vouching". These are trust scores that we use to initialize (and more) the EigenTrust algorithm (vouching)

Reply:
OK I see, thank you. If these are trust scores used to initialized the algorithm how about pre-trust scores? or initial trust? (or initial network trust?)

I'm OK with the current formulation in fact, I wanted to be sure that it was a different concept than pre-trust users. If no it was the good moment to change the comment.